test/multiverse/suites/curb/Envfile

```diff
@@ -40,13 +40,4 @@ gemfile <<~RB
   gem 'curb', '#{curb_gem_version}'
   gem 'rack'
   gem 'json', :platforms => [:rbx, :mri_18]
-
-  
-RB
-
-gemfile <<~RB
```

Comment:
This one was just a exact duplicate of the one above, so we were running all the curb tests twice for the same version

Reply:
:sweat_smile: thanks for catching this!